lib/promscrape/targets_response.qtpl

```diff
@@ -0,0 +1,90 @@
+{% import "github.com/VictoriaMetrics/VictoriaMetrics/lib/prompbmarshal"
+%}
+
+{% func TargetsResponsePlain (jts []jobTargetsStatuses,  showOriginLabels bool) -%}
```

Comment:
It is better from maintainability PoV to wrap all the functions into `{% stripspace %} ... {% endstripspace %}` and explicitly insert `{% space %}` with `{% newline %}` where necessary instead of adding `-` for every template tag. See https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/app/vmselect/prometheus/export.qtpl for details.

Reply:
done, thx!